src/aiohomeconnect/cli/__init__.py

```diff
@@ -119,8 +136,17 @@ async def _subscribe_appliance_events(
 ) -> None:
     """Subscribe and print events from one appliance."""
     client = CLIClient(client_id, client_secret)
-    async for event in client.stream_events(ha_id):
-        rich_print(event)
+    try:
+        while True:
+            try:
+                async for event in client.stream_events(ha_id):
+                    rich_print(event)
+            except ReadTimeout as e:
+                rich_print(f"{e} continuing...")
+            except RemoteProtocolError as e:
+                rich_print(f"{e}, continuing...")
+    except HomeConnectError as e:
```

Comment:
We can put this on the same level as the other and `break` the loop for this exception.

Reply:
Should we break the loop?
I mean, for this client which is for testing propourses, maybe, but for implementation such Home Assistant we should continue as it isn't a breaking error, because you can re-request the stream and continue working.

Reply:
I was only referring to the two try... except statements here and indentation which isn't needed if we use the `break` statement in the loop.